experimental/traffic-portal/src/app/api/coordinate.service.spec.ts

```diff
@@ -79,6 +79,38 @@ describe("CoordinateService", () => {
 		await expectAsync(responseP).toBeResolvedTo(coordinate);
 	});
 
+	it("sends requests for multiple coordinates by ID", async () => {
```

Comment:
These `it`s make it seem like the service should be doing something successfully. We're actually testing that it throws an error when a request that's supposed to only return one thing instead returns many.